src/libLookup/Lookup.cpp

```diff
@@ -117,19 +119,41 @@ void Lookup::SendMessageToLookupNodes(
                                                 << ":"
                                                 << node.m_listenPortHost);
 
-        AllLookupNodes.push_back(node);
+        allLookupNodes.push_back(node);
     }
 
-    P2PComm::GetInstance().SendBroadcastMessage(AllLookupNodes, message);
+    P2PComm::GetInstance().SendBroadcastMessage(allLookupNodes, message);
```

Comment:
Can you use  m_lookup->SendMessageToLookupNodes(message) ?

Is using broadcast safe here?

Reply:
This function is mean to do so, it's only used when intent. However I didn't change the code here except for the naming of AllLookupNodes->allLookupNodes

Reply:
This function is used by non-lookup nodes including the broadcasting of newly created DSBlock, sharding structure, final block

Reply:
ok